getting-started/vs-code-integration/new-project-wizard.md

```diff
@@ -25,9 +25,11 @@ To create a Telerik-enabled Blazor project:
 
 1. Type/Select `Telerik UI for Blazor Template Wizard: Launch` and press `Enter` to launch the extension.
 
-## Creating the Project
+## Using the Wizard
 
-The **Create New Project** wizard provides two options to start you project:
+### Section 1: Choose how to start the project
```

Comment:
The letter casing is inconsistent between the steps. Some use sentence case and some use first capital letter for each word.